lib/capistrano/tasks/docker.rake

```
@@ -35,21 +45,25 @@ namespace :docker do
   task :build => [:'docker:update_mirror'] do
     on fetch(:docker_build_server_host) do
       within dockerbuild_plugin.docker_build_base_path do
-        timestamp = Time.now.to_i
-        git_sha1 = `git rev-parse #{fetch(:branch)}`.chomp
-        worktree_dir_name = "worktree-#{git_sha1}-#{timestamp}"
+        if fetch(:docker_build_no_worktree)
+          execute(:git, :reset, "--hard", fetch(:branch))
+          execute(*fetch(:docker_build_cmd))
+        else
+          timestamp = Time.now.to_i
+          worktree_dir_name = "worktree-#{fetch(:docker_tag)}-#{timestamp}"
```

I think it is better to keep using `git_sha1` because users can set an arbitrary value to `docker_tag` such as "latest", and they might face the error "fatal: 'foo' already exists."